generated/nidaqmx/system/watchdog.py

```diff
@@ -188,8 +186,8 @@ def expir_trig_dig_edge_src(self, val):
             self._handle, val)
         check_for_error(error_code)
 
-    @expir_trig_dig_edge_src.deleter
-    def expir_trig_dig_edge_src(self):
+    @dig_edge_watchdog_expir_trig_src.deleter
+    def dig_edge_watchdog_expir_trig_src(self):
         cfunc = lib_importer.windll.DAQmxResetDigEdgeWatchdogExpirTrigSrc
         if cfunc.argtypes is None:
             with cfunc.arglock:
```

Comment:
I can't make a comment on the network_conn_loss, but I think we should change it...

`expiration_trigger_trig_on_network_conn_loss`